src/main/java/ludum/dare/components/upgradeComponents/MetalComponent.java

```diff
@@ -13,7 +13,7 @@
 
     public MetalComponent(PhysicsComponent phys, HealthComponent health, AttackComponent attack){
         phys.getBody().props.acceleration = (int) (phys.getBody().props.acceleration * 0.9f);
-        phys.getBody().props.gravityModifier = phys.getBody().props.gravityModifier * 0.9f;
+        phys.getBody().props.gravityModifier = phys.getBody().props.gravityModifier * 1.1f;
```

Comment:
careful with multiplication in the case we get multiple things modifying the same stat. Adding is a better route so we know we can undo exactly what we did (ex. + 10 / -10 will always work. * 1.1 / * 0.9 won't always do what you expect